tests/test_humanize.py

```diff
@@ -47,6 +47,13 @@ def test_naturaltime(self, h):
         assert h._humanize(now) == 'now'
         assert h._humanize(now - one_month) == 'a month ago'
 
+    def test_naturaltime_with_utc(self, h, app):
+        app.config['HUMANIZE_USE_UTC'] = True
+        h.init_app(app)
+        assert  h._humanize(datetime.utcnow()) == 'now'
+        app.config['HUMANIZE_USE_UTC'] = False
+        h.init_app(app)
+
```

Comment:
This test suite can be rewritten with some neat pytest-flask features:

- Pass options via [`pytest.mark.options` decorator](http://pytest-flask.readthedocs.org/en/latest/features.html#pytest-mark-options-pass-options-to-your-application-config)

- No need to initialize extension explicitly, it will be initialized by `pytest` during resolving fixture

- Refine test suite, e.g. split test suites into different functions (one test per feature) and move them into separate namespace for better readability

This test suite looks good for me:

```python
@pytest.mark.usefixtures('app')
class TestUTC:

    @pytest.mark.options(humanize_use_utc=True)
    def test_use_utc(self, h):
        assert humanize.time._now == datetime.utcnow

    @pytest.mark.options(humanize_use_utc=False)
    def test_dont_use_utc(self, h):
        assert humanize.time._now == datetime.now

    @pytest.mark.options(humanize_use_utc=True)
    def test_naturaltime_with_utc(self, h):
        assert h._humanize(datetime.utcnow()) == 'now'
```